src/options/parser.rs

```diff
@@ -1,748 +1,217 @@
+    #[arg(short = 't', long)]
+    pub time: Option<OsString>,
+    /// use the accessed timestamp field.
+    #[arg(short = 'u', long, action = clap::ArgAction::Count)]
+    pub accessed: u8,
+    /// use the created timestamp field.
+    #[arg(short = 'U', long, action = clap::ArgAction::Count)]
+    pub created: u8,
+    /// how to format timestamps (default, iso, long-iso, full-iso, relative).
+    #[arg(long = "time-style")]
+    pub time_style: Option<OsString>,
+    /// display entries as hyperlinks.
+    #[arg(long, action = clap::ArgAction::Count)]
+    pub hyperlink: u8,
+    /// supress the permissions field.
+    #[arg(long = "no-permissions", action = clap::ArgAction::Count)]
+    pub no_permissions: u8,
+    /// suppress the filesize field.
+    #[arg(long = "no-filesize", action = clap::ArgAction::Count)]
+    pub no_filesize: u8,
+    /// suppress the user field.
+    #[arg(long = "no-user", action = clap::ArgAction::Count)]
+    pub no_user: u8,
+    /// suppress the time field.
+    #[arg(long = "no-time", action = clap::ArgAction::Count)]
+    pub no_time: u8,
+    /// don't display icons (always override --icons).
+    #[arg(long = "no-icons", action = clap::ArgAction::Count)]
+    pub no_icons: u8,
+    /// supress git.
+    #[arg(long = "no-git", action = clap::ArgAction::Count)]
+    pub no_git: u8,
+    /// list root of git-tree status.
+    #[arg(long = "git-repos", action = clap::ArgAction::Count)]
+    pub git_repos: u8,
     ///
-    /// It’s not possible to tell which flag the value belonged to from this.
-    pub fn get_where<P>(&self, predicate: P) -> Result<Option<&OsStr>, OptionsError>
-    where P: Fn(&Flag) -> bool {
-        if self.is_strict() {
-            let those = self.flags.iter()
-                            .filter(|tuple| tuple.1.is_some() && predicate(&tuple.0))
-                            .collect::<Vec<_>>();
-
-            if those.len() < 2 { Ok(those.first().copied().map(|t| t.1.unwrap())) }
-                          else { Err(OptionsError::Duplicate(those[0].0, those[1].0)) }
-        }
-        else {
-            let found = self.flags.iter().rev()
-                            .find(|tuple| tuple.1.is_some() && predicate(&tuple.0))
-                            .map(|tuple| tuple.1.unwrap());
-            Ok(found)
+    #[arg(long = "git-repos-no-status", action = clap::ArgAction::Count)]
+    pub git_repos_no_status: u8,
+    /// list each file's permission in octal format.
+    #[arg(short = 'o', long, action = clap::ArgAction::Count)]
+    pub octal: u8,
+    /// Display the number of hard links to file.
+    #[arg(short = 'Z', long = "context", action = clap::ArgAction::Count)]
+    pub security_context: u8,
+    /// Show extended attributes.
+    #[arg(short = '@', long, action = clap::ArgAction::Count)]
+    pub extended: u8,
+    /// show list of command-line options.
+    #[arg(short ='?', long, action = clap::ArgAction::Help)]
+    pub help: Option<bool>,
```

Comment:
If it is a bool it is made required by clap

Reply:
Yes but I think that giving it the type option is better cause it make it more clear for me

Reply:
Looking at the code some more, why do you use `#[clap(disable_help_flag = true)]` then reintroduce it manually?

Reply:
It is because the base help of clap is forcing `-h` as a flag for help or we need it for `--header` so I had to reintroduce it like that.

Reply:
> It is because the base help of clap is forcing `-h` as a flag for help or we need it for `--header` so I had to reintroduce it like that.

perhaps we should prefer `-h` for help, but this should be marked as a breaking change as described in https://www.conventionalcommits.org/en/v1.0.0/

Reply:
Well this need a debate and maybe more talk because the -h is used in many aliases and all. I also understand that it is now an industry standard to have it for help but it is also something here since a long time in exa/eza.

Reply:
Ah yes indeed, but then it should probably explained in a comment (in a simple sentence like `// disabled so that we can use -h as a short flag for --header`).

Reply:
well as of the conversation we had on gitter we decided to put help back on -h because it is an industry standard.